packages/localize/test/LocalizeManager.test.js

```diff
@@ -116,6 +116,25 @@ describe('LocalizeManager', () => {
       });
     });
 
+    it('can load a namespace for a different locale', async () => {
+      setupFakeImport('./my-component/nl-NL.js', { default: { greeting: 'Hello!' } });
+
+      const manager = new LocalizeManager();
+
+      await manager.loadNamespace(
+        {
+          'my-component': locale => fakeImport(`./my-component/${locale}.js`),
+        },
+        'nl-NL',
```

Comment:
I would prefer to keep this consistent with `msg` and use an opts object here with `{ locale: 'nl-NL' }`. Not that this will necessarily be needed, but I remember that we wanted to extend options later with extra features and hence made it an object in `msg`, and I'm afraid that people will automatically expect this APIs to be mirrored.

Besides that I like the MR. I will update it accordingly myself and check with my team if they agree with my proposed API change.

Reply:
Sounds good to me!